### ISSUE TITLE
Error when installing the package

### ISSUE THREAD

**Issue by NotSoShaby** [PetrDlouhy/django-admin-charts#36]
I'm getting a pretty general error when installing the package.

I installed all requierments and followef installation process. 

When installed apps:
```
INSTALLED_APPS = [
...
    # 3rd party apps
    # 'admin_tools_stats',
    'django_nvd3',
...]
```

log:
serverWarmup ✘ refael@Refael-laptop  ~/Desktop/serverWarmup  python serverWarmup/manage.py check --settings=serverWarmup.dev_settings
System check identified no issues (0 silenced).

When installed apps: 
```
INSTALLED_APPS = [

...

    # 3rd party apps
    'admin_tools_stats',
    # 'django_nvd3',
...]
```
Log
```
serverWarmup @-laptop  ~/Desktop/serverWarmup  python serverWarmup/manage.py check --settings=serverWarmup.dev_settings
Traceback (most recent call last):
  File "/home/refael/Desktop/serverWarmup/serverWarmup/manage.py", line 22, in <module>
    main()
  File "/home/refael/Desktop/serverWarmup/serverWarmup/manage.py", line 18, in main
    execute_from_command_line(sys.argv)
  File "/home/refael/.local/share/virtualenvs/serverWarmup-6xk3Qqhq/lib/python3.9/site-packages/django/core/management/__init__.py", line 425, in execute_from_command_line
    utility.execute()
  File "/home/refael/.local/share/virtualenvs/serverWarmup-6xk3Qqhq/lib/python3.9/site-packages/django/core/management/__init__.py", line 401, in execute
    django.setup()
  File "/home/refael/.local/share/virtualenvs/serverWarmup-6xk3Qqhq/lib/python3.9/site-packages/django/__init__.py", line 24, in setup
    apps.populate(settings.INSTALLED_APPS)
  File "/home/refael/.local/share/virtualenvs/serverWarmup-6xk3Qqhq/lib/python3.9/site-packages/django/apps/registry.py", line 83, in populate
    raise RuntimeError("populate() isn't reentrant")
RuntimeError: populate() isn't reentrant

```

I tried a all the suggestions on [this](https://stackoverflow.com/questions/27093746/django-stops-working-with-runtimeerror-populate-isnt-reentrant) SO question, but none works

EDIT:

The issue was with the smart_text import which no longer exist/works in django. No solution for now, need to fork, change and create a PR.

Possible a simple self-made function like suggested [here](https://stackoverflow.com/questions/38029253/is-there-a-way-to-replace-djangos-smart-str-with-a-python-native-solution)

**Comment by PetrDlouhy** [comment]
Hi @NotSoShaby,

which version of Django and `django-admin-charts` do you use?
It seems odd, that this would be caused by smart_text error in `django-admin-charts`, because the source is tested with Django 2.2 - 4.0 by automatic tests, and I am using the app with Django 4.0.

I guess, the problem might be in some other application or in combination of more applications.

**Comment by PetrDlouhy** [comment]
I have released new version 1.2.0, which fixes problem with `django-cache-utils`.
@NotSoShaby I am closing this issue, please reopen this or open new issue, if there are any problems with the new version.
